build-system/global-configs/prod-config.json

```diff
@@ -40,6 +40,6 @@
   "expAdsenseCanonical": 0,
   "font-display-swap": 1,
   "amp-date-picker": 1,
-  "url-replacement-v2": 0,
+  "url-replacement-v2": 0.5,
```

Comment:
let's do 1 for this time. We've been on .5 for a while and no other issues reported.

Reply:
Done. Even used the github.com editor :)